flask-server/create_db.py

```diff
@@ -1,22 +1,27 @@
-# Run this file only if database is not yet created
 from dotenv import load_dotenv
 import os
 import mysql.connector
-# from mysqlconfig import HOST, USER, PASSWORD
 
-load_dotenv()
+load_dotenv(override=True)
 
 mydb = mysql.connector.connect(
     host=os.getenv("HOST"),
     user=os.getenv("USER"),
     passwd=os.getenv("PASSWORD"),
 )
 
+database = os.getenv("DATABASE")
+test_database = os.getenv("TESTDB")
 my_cursor = mydb.cursor()
 
-my_cursor.execute("CREATE DATABASE introverse_dev")
-
-# Can run this bit below too if you want to see all the databases in your MySQL (and will confirm creation of new one)
-my_cursor.execute("SHOW DATABASES")
-for db in my_cursor:
-    print(db)
\ No newline at end of file
+def create_db_if_not_exist(dbname):
```

Comment:
If they already exist it will print that they exist. Maybe I should rename the function? "Check if exist and create if not" or something.
![image](https://github.com/Angel2001-programmer/Introverse/assets/130995689/cf8ab1a9-b1f0-4208-8b84-2407e820784c)

Reply:
Have renamed function to make its purpose more clear